app/assets/javascripts/initializers/initializeOnboardingTaskCard.js

```diff
@@ -11,10 +11,15 @@ function initializeOnboardingTaskCard() {
 
   var taskCard = document.getElementsByClassName('onboarding-task-card')[0];
   if (taskCard == null) {
+    return; // This guards against both a null and undefined taskCard.
+  }
+
+  const user = userData();
+  if (!user) {
```

Comment:
Can we not just move this into an or statement, i.e. :
```
var taskCard = document.getElementsByClassName('onboarding-task-card')[0];
const user = userData();
if (taskCard == null ||  !user) {
  return;
}
```

Reply:
We can -- I clearly had a fried brain when I wrote this last night 😓 Updated in the latest commit!